documentation/onboard-service.md

```diff
@@ -100,6 +100,8 @@ The code deploys integrated DNS zone and Azure front door **in the production su
 ### Deploy infrastructure
 This deploys one DNS zone and one front door which will be used subsequently by all the domains.
 
+Refer to the new service template in templates/new_service/terraform/domains/infrastructure, for the files needed before running the commands below:
```

Comment:
At this point, users will have already used the template script as per https://github.com/DFE-Digital/teacher-services-cloud/blob/main/documentation/onboard-service.md#generate-code
So there is no template involved anymore.

Reply:
Ok, maybe I'd remove - It probably applies or help someone stating from scratch. Perhaps I could keep but add something like "Ensure the files present in /terraform/domains/infrastructure matches the sample structure in templates/new_service/terraform/domains/infrastructure, with the appropriate configuration" ?

Reply:
As discussed, move to the top of the Custom domain section with more generic "Ensure..."